gen-webpack-config.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
```

Comment:
🛑 There should probably be a comment in this file explaining how this is used and where.

🛑 Also, it should probably be moved to the `utils/` directory with the other gen script.